pallets/sp-mvm/src/lib.rs

```diff
@@ -220,17 +224,37 @@ pub mod pallet {
             module_bc: Vec<u8>,
             gas_limit: u64,
         ) -> DispatchResultWithPostInfo {
-            let who = ensure_signed(origin)?;
-            debug!("executing `publish` with signed {:?}", who);
+            // Allows to update Standard Library if root.
+            let (sender, signer) = match T::UpdaterOrigin::ensure_origin(origin.clone()) {
+                Ok(_) => {
+                    debug!("executing `publish module` with root");
+                    (CORE_CODE_ADDRESS, sudo::Pallet::<T>::key())
```

Comment:
Remove sudo, you really don't need it and it's not correct way.
Instead use: `Ok(signer)` instead of `Ok(_)`